thoth/package_analyzer/python/fetch.py

```diff
@@ -38,11 +39,22 @@ def fetch(self, package_name: str, package_version: str) -> dict:
             "Fetching digests for package %r in version %r from %r", package_name, package_version, self.source.url
         )
 
-        artifacts = self.source.get_package_hashes(package_name, package_version, True)
+        error = False
+        error_message = ""
+        artifacts = []
+
+        try:
+            artifacts = self.source.get_package_hashes(package_name, package_version, True)
+        except NotFound as exc:
+            error = True
+            error_message = str(exc)
+            _LOGGER.warning(error_message)
```

Comment:
Logging as a warning instead of as an exception.

Reply:
Good idea, the warning is sufficient enough and we don't get these in the reporting system :+1: